autopush/websocket.py

```diff
@@ -1215,6 +1215,8 @@ def process_register(self, data):
             d = self.deferToThread(self.conf.make_endpoint,
                                    self.ps.uaid, chid, data.get("key"))
         else:
+            if self.conf.disable_simplepush:
+                return self.bad_message("register", "No longer supported")
```

Comment:
Ok, I see, my bad, I was under the impression we dropped them at hello if they weren't using webpush. Why not just drop at hello?